app/src/main/java/com/example/admin/learningandroid/fragments/FragmentActivity.java

```diff
@@ -0,0 +1,41 @@
+package com.example.admin.learningandroid.fragments;
+
+import android.support.v4.app.FragmentManager;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+
+import com.example.admin.learningandroid.R;
+
+
+public class FragmentActivity extends AppCompatActivity implements PassingDataFromSendingFragmentToReceivingFragment,PassingDataFromReceivingFragmentToSendingFragment {
```

Comment:
I think the names should be shorter and crisper and should follow the conventions

Reply:
The interface can have two methods. The name of the interface can be something like FragmentInterface and it can have two methods like setData and clearData